src/lambda_function.py

```diff
@@ -110,6 +115,25 @@ def lambda_handler(event, context):
     else:
         object['results'].append({'type': 'bias', 'outcome': {"error" : "The bias score was not available."}})
 
+    # SpacyMatcher
+    logger.info(f'LambdaFunction: Trying to get Spacy Matcher...')
+    if sentanalysisresult['text'] != '-1':
+        try:
+            dict_tot = {}
+            tags = ['PERSON','GPE','ORG','PERCENT','LANGUAGE','DATE','TIME','LOC','NORP','EVENT','WORK_OF_ART',
+                    'MONEY','QUANTITY','ORDINAL','CARDINAL']
+            for i in tags:
+                list_objs = sm.spacyMatcher(sentanalysisresult['text'], i) ### {'Toyota', 'BBC', 'BMW', 'Chanel'}
+                for obj in list_objs:
+                    dict_tot[obj] = i                    
+            object['results'].append({'type': 'topics', 'outcome': dict_tot})
```

Comment:
Can I suggest we move this to the 'article' object, and bundle each topic/type as a dict?

```
article_topics = []
for i in tags:
                list_objs = sm.spacyMatcher(sentanalysisresult['text'], i) ### {'Toyota', 'BBC', 'BMW', 'Chanel'}
                for obj in list_objs:
                    article_topics.append({'type' : i, 'topic' : obj})

object['article']['topics'] = article_topics
```

...which should return an object like:

```
{
  "url": ...
  "results": ...
  "article": {
    "header":...
    "topics":[
      {"type":"PERSON", "topic":"Boris Johnson"},
      {"type":"GPE","topic":"United Kingdom"}
    ]
  }
}
```

Reply:
I think this'll be easier to process on the frontend, as we can do a .map() on the topics list, and then pass the topic object (`{"type":"PERSON", "topic":"Boris Johnson"}`) to the chip to display.

Reply:
That is not a problem ...! 
Can I please ask you to first fix the error: "E       OSError: [E050] Can't find model 'en_core_web_sm'. It doesn't seem to be a shortcut link, a Python package or a valid path to a data directory."
I guess we need to install the following module in AWS:
=> python -m spacy download en_core_web_sm

Reply:
I've just made a commit (e86d604) which add's the install command to the build_script.sh... just need to see if it works! :)